app/assets/javascripts/dashboard.js

```diff
@@ -1,32 +1,57 @@
-//= require leaflet.locate
-
 $(function () {
   const defaultHomeZoom = 12;
   let map;
 
   if ($("#map").length) {
-    map = L.map("map", {
+    map = new maplibregl.Map({
+      container: "map",
+      style: {
+        version: 8,
+        name: "OSM Raster",
+        sources: {
+          osm: {
+            type: "raster",
+            tiles: [
+              "https://tile.openstreetmap.org/{z}/{x}/{y}.png"
+            ],
+            tileSize: 256,
+            maxzoom: 19,
+            attribution: "© OpenStreetMap contributors"
```

Comment:
Do we need to set this property if `attributionControl` is `false`?

Reply:
I guess for now this can be removed, but it will be re-added in the next PR where this is generalised.

I don't think this matters.

```suggestion
```